src/project/settings/base.py

```diff
@@ -36,9 +38,20 @@
     'geostore',
     'django_geosource',
     'terra_layer',
+    'mapbox_baselayer',
+    'django.contrib.admin',
+    'django.contrib.messages',
     'custom.dataloader',
+    'django_json_widget',
+    'reversion',
+    'sorl.thumbnail',
```

Comment:
'mapbox_baselayer',
'django_json_widget',
'reversion',
'sorl.thumbnail',

are specific to geocrud, move to "if USE_TERRA_GEOCRUD"